core/client/src/test/java/alluxio/client/file/FileOutStreamTest.java

```diff
@@ -97,11 +104,13 @@
   @Before
   public void before() throws Exception {
     ClientTestUtils.setSmallBufferSizes();
+    mDelegationEnabled = Configuration.getBoolean(PropertyKey.USER_UFS_DELEGATION_ENABLED);
```

Comment:
I didn't find an easy way to use Configuration Rule and support both versions, are you okay with the current implementation?